config/consensus.go

```diff
@@ -421,6 +421,11 @@ type ConsensusParams struct {
 	// The hard-limit for number of StateProof keys is derived from the maximum depth allowed for the merkle signature scheme's tree - 2^16.
 	// More keys => deeper merkle tree => longer proof required => infeasible for our SNARK.
 	MaxKeyregValidPeriod uint64
+
+	// EnableSHA256TxnRootHeader enables a new header field: the transaction merkle tree's root using SHA256 (actually vector commitment and not merkle tree).
+	// This new header is in addition to the existing SHA512_256 merkle root.
```

Comment:
1- I think we should just mention that this is a vector commitment root. it should be clear what it is as we are using it quite often.
2- I would recommend not to couple this field with stateproof. maybe we can just mention that "this might be useful to verify transactions on our blockchains"